atom/browser/api/atom_api_screen.cc

```diff
@@ -107,7 +122,8 @@ mate::ObjectTemplateBuilder Screen::GetObjectTemplateBuilder(
       .SetMethod("getPrimaryDisplay", &Screen::GetPrimaryDisplay)
       .SetMethod("getAllDisplays", &Screen::GetAllDisplays)
       .SetMethod("getDisplayNearestPoint", &Screen::GetDisplayNearestPoint)
-      .SetMethod("getDisplayMatching", &Screen::GetDisplayMatching);
+      .SetMethod("getDisplayMatching", &Screen::GetDisplayMatching)
+      .SetMethod("getDesktopCapturer", &Screen::GetDesktopCapturer);
```

Comment:
I have updated the design of `desktop_capturer` module as we discussed. It is ready for review now.